packages/frontend/src/components/Form/Form.tsx

```diff
@@ -20,3 +20,16 @@ export const FormRow = styled.div`
 export const FormHeading = styled.h2`
   color: ${Colors.White};
 `
+
+export const FormWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  row-gap: 16px;
+  margin-left: -170px;
+  padding: 82px 115px;
+  position: relative;
+  z-index: 100;
+  width: 724px;
+  height: 450px;
+  background: #6100ff;
+`
```

Comment:
```suggestion
export const FormWrapper = styled.div`
  display: flex;
  flex-direction: column;
  row-gap: 16px;
  padding: 82px 115px 82px 170px;
  width: 100%;
`
```

I would suggest to extract form styles separately from section, because forms will be different. 
I add some [common wrapper](https://github.com/TrueFiEng/devcon-raffle/blob/5a2016d5774156cd801e6d69bbcb91f501557937/packages/frontend/src/components/Auction/AuctionAction.tsx) in #29, so it can be changed later

Reply:
Can you verify if we applied your suggestion?